tests/assertions.js

```diff
@@ -30,10 +30,7 @@ export const verificationFail = async ({
   should.not.exist(result, reason);
   should.exist(error, 'Expected verifier to error.');
   should.exist(error.status, 'Expected verifier to return an HTTP Status code');
-  error.status.should.equal(
-    400,
-    'Expected HTTP Status code 400 invalid input!'
-  );
+  error.status.should.be.oneOf([400, 422], reason);
```

Comment:
Should this `reason` be something closer to the original rather than "Expected no result from verifier."?

Reply:
hmmmm... I honestly was just thinking about the single case here where Greg's implementation runs 500 and I didn't know which test vector was causing that, so I think having a different default makes sense and I'll make that change momentarily. good idea.

Reply:
added the additional default here https://github.com/w3c/vc-di-bbs-test-suite/pull/49/commits/f1fbfad829c6221eb394380662ee0468b59cbf5a